lib/l10n/app_de.arb

```diff
@@ -1,179 +1,178 @@
+  "disableLogsSharingText": "Gemeinsame Nutzung der Anrufliste deaktivieren",
+  "donateText": "Spenden",
+  "downloadConfirmationText": "Sind Sie sicher, dass Sie Ihre Anrufprotokolle im Format {currentImportType} herunterladen möchten? Mit dieser Aktion wird Ihr Anrufprotokoll in einer Datei {currentImportType} auf Ihrem Gerät gespeichert.",
+  "downloadText": "Downloaden",
+  "dtAAcronymExapandedText": "Vollständiger Name des Wochentags",
+  "dtBAcronymExapandedText": "Vollständiger Name des Monats",
+  "dtHAcronymExapandedText": "Uhrzeit im 24-Stunden-Format",
+  "dtIAcronymExapandedText": "Uhrzeit im 12-Stunden-Format",
+  "dtMAcronymExapandedText": "Minuten",
+  "dtSAcronymExapandedText": "Sekunden",
+  "dtUAcronymExapandedText": "Wochennummer des Jahres (Sonntag als erster Tag der Woche)",
+  "dtWAcronymExapandedText": "Wochennummer des Jahres (Montag als erster Tag der Woche)“,
+  "dtXAcronymExapandedText": "Zeitliche Darstellung",
+  "dtYAcronymExapandedText": "Jahr mit Jahrhundert",
+  "dtZAcronymExapandedText": "Name der Zeitzone",
+  "dtaAcronymExapandedText": "Abgekürzter Name des Wochentags",
+  "dtbAcronymExapandedText": "Abgekürzter Monatsname",
+  "dtcAcronymExapandedText": "Darstellung von Datum und Uhrzeit",
+  "dtdAcronymExapandedText": "Tag des Monats",
+  "dtfAcronymExapandedText": "Millisekunden",
+  "dtjAcronymExapandedText": "Tag des Jahres",
+  "dtmAcronymExapandedText": "Monat als Nummerr",
+  "dtpAcronymExapandedText": "AM/PM-Markierung",
+  "dtxAcronymExapandedText": "Darstellung des Datums",
+  "dtyAcronymExapandedText": "Jahr ohne Jahrhundert",
+  "dtzAcronymExapandedText": "Zeitzonenabweichung von UTC",
+  "durationText": "Dauer",
+  "enableCallDurationFilteringText": "Dauerfilterung einschalten",
+  "enableCallLogCountVisibilityText": "Anzahl der Anrufprotokolle anzeigen",
+  "enableFilterByAccountIdText": "Filter nach Konto-ID aktivieren",
+  "exploreNowText": "Jetzt erkunden",
+  "exploreProjectButtonText": "Projekt auf Github erkunden",
+  "exportFileNameFormatLabelText": "Dateinamenformat exportieren",
+  "filenameFormatHintText": "mein-anruf-protokoll-[%token]",
+  "filenameFormatLabelText": "Dateinamen-Format",
+  "filterText": "Filter",
+  "getStartedText": "Los geht's",
+  "ghostErrorMessage": "Hmm. Etwas ist schief gelaufen",
+  "hideText": "Ausblenden",
+  "hintMobileNumberText": "9XXXX XXXX",
+  "importCallLogsText": "Importieren Sie Ihre Anrufprotokolle",
+  "incomingText": "Eingehende",
+  "logsScreenLabelText": "Protokolle",
+  "maximumDurationHintText": "bsp: 60",
+  "maximumDurationLabelText": "Min (in secs)",
+  "minimumDurationHintText": "bsp: 0",
+  "minimumDurationLabelText": "Min (in secs)",
+  "missedText": "Verpasst",
+  "mobileNumberText": "Handynummer",
+  "nextText": "Weiter",
+  "onboardingScreenFourSubtitle": "Übertragen Sie Anrufprotokolle nahtlos zwischen Geräten oder exportieren Sie sie zur sicheren Aufbewahrung und Analyse.",
+  "onboardingScreenFourTitle": "Import & Export",
```

Comment:
This doesn't seems to be in German ? Google suggests me : 

```json
"onboardingScreenFourTitle:" "Importieren und Exportieren"
```
Please correct me if i am wrong. Really appreciate your work here

Reply:
In german both, "Immportieren und exportieren", "import und Export " is the same